roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -32,19 +32,19 @@ mon host = {% for host in groups[mon_group_name] %}{{ hostvars[host]['ansible_'
 {% if mon_containerized_deployment %}
 fsid = {{ fsid }}
 {% if groups[mon_group_name] is defined %}
-{% for host in groups[mon_group_name] %}
+mon host = {% for host in groups[mon_group_name] -%}
```

Comment:
Before these changes we were generating the `ceph.conf` `mon host` variable as:
```
mon_host = XXX.XXX.XXX.XXX
,mon_host = XXX.XXX.XXX.XXX
,mon_host = XXX.XXX.XXX.XXX
```

Now we generate it as:
```
mon host = XXX.XXX.XXX.XXX,XXX.XXX.XXX.XXX,XXX.XXX.XXX.XXX
```